src/app/fdctl/utility.c

```diff
@@ -14,21 +14,37 @@
 #include <linux/netlink.h>
 #include <linux/rtnetlink.h>
 
-static int entered = 0;
+static int namespace_original_fd = 0;
 
 void
-enter_network_namespace( config_t * const config ) {
-  if( FD_LIKELY( !config->development.netns.enabled || entered ) ) return;
```

Comment:
Why did we remove this check? I think it's an error to enter the namespace if netns is not enabled, that will prevent bind from succeeding on a host device.

Reply:
that was a mistake good catch